codota/tabnine-intellij
# DEV2-1681 - add "Getting Started" page

**Status bar changes:**
The previous status bar widget was a pure swing component that was very hard to manage, and the popup wasn't supported.
I changed it to extend `StatusBarWidget.MultipleTextValuesPresentation` instead of the `CustomStatusBar` so we can support the popup easily, add a separate method that gets the icon, and a separate method that gets the text.

**Open the getting-started page:**
I used CEF, which is chromium for java. the issue is that only IJ version 20.2 and later support this feature, so I used reflection to get over the compilation error for the older versions.

**QA:**

Checked that the popup and the buttons work properly:
https://user-images.githubusercontent.com/10290661/202397396-db818774-68b6-4c2f-9b3d-281d88f1dc56.mp4

Checked that when there are **no** local binary versions, the getting-started page should open on IDE startup:
https://user-images.githubusercontent.com/10290661/202398011-a533f138-d3b7-4157-ae46-4029ec034232.mp4

Checked that when there are local binary versions, the getting-started page should not open on IDE startup:
https://user-images.githubusercontent.com/10290661/202398252-1bef4d53-84d4-45d5-b2e8-f40370bb9dac.mp4
